### DISCUSSION TITLE
Are there Open Source projects using Rocket?

### DISCUSSION THREAD

**Discussion by szabgab** [rwf2/Rocket#2705] [Questions]
Is there a list of Open Source project using Rocket?

If you are writing any project using Rocket where the source code is public, please comment with the URL.

**Comment by wezm** [comment]
These little projects of mine use it:

- [linkedlist.org](https://linkedlist.org/) - Personal Knowledgebase ⟡ [source](https://github.com/wezm/pkb)
- [dewpoint.7bit.org](https://dewpoint.7bit.org/) - Dewpoint weather forecasts ⟡ [source](https://github.com/wezm/dewpoint.7bit.org)
- [quotes.randome.net](https://quotes.randome.net/) - Quotes between friends using UI I designed in 2003 ⟡ [source](https://github.com/wezm/Quotes)

**Comment by olback** [comment]
I only have one open-source project that uses Rocket atm:
[tempfiles.ninja](https://tempfiles.ninja/) - [source](https://github.com/olback/tempfiles-rs) - A simple file sharing service that encrypts files at rest  and deletes them automatically after 24 hours.

**Reply by wezm** [reply]
Oh this is cool. Nice one.

**Reply by SergioBenitez** [reply]
There's also a short list here: https://rocket.rs/v0.5/guide/faq/#showcase

**Comment by Kapsyloffer** [comment]
I am in the middle of writing a lichess clone of [SHOBU](https://www.ultraboardgames.com/shobu/game-rules.php) in it. 💪  
The plan is that I should be able to challenge my friends online with a privately generated lobby, nothing super big scale yet. If all goes well I might implement ELO and look into making the "lobby system" actually good.

https://github.com/Kapsyloffer/RUSTBU

It is basically 30% done, still need to do a frontend and other things. I am still in the write functions and unittest -stage, but soon I'll be able to throw something together. (hopefully)

**Comment by SergioBenitez** [comment]
There's a short list here: https://rocket.rs/v0.5/guide/faq/#showcase
